src/howTos/dev/services.md

```diff
@@ -111,20 +105,19 @@ Each time you make modifications to your service, you must update the app on the
 cozy-stack apps update <app_name>
 ```
 
-
 ## Execution
 
 The service will be run each time the trigger condition is met, e.g. a bank operation.
-However, you can force its execution thanks to the `cozy-run-dev` CLI.
+However, you can force its execution thanks to the `cozy-run-dev` CLI, which can be useuful for developement.
 
-To install:
+To install locally:
 ```bash
-yarn global add cozy-jobs-cli
+yarn add --dev cozy-jobs-cli
 ```
 
 To run:
 ```bash
-cozy-konnector-dev -m <app_manifest> <mybuiltservice.js>
+./node_modules/.bin/cozy-konnector-dev -m <app_manifest> <mybuiltservice.js>
```

Comment:
De mon côté j'ai ajouté une règle dans package.json :
`"services:checkAndUpdate": "cozy-konnector-dev -m manifest.webapp build/services/checkAndUpdate/contacts.js"`

Peut-être ajouter à la doc, ça éviter de lancer des commandes trop longue ?

Reply:
npm run can also be used.

```patch
- ./node_modules/.bin/cozy-konnector-dev -m <app_manifest> <mybuiltservice.js>
+ npm run cozy-konnector-dev -m <app_manifest> <mybuiltservice.js>
```

Reply:
Nice tip for the run command, I added it with yarn, as we tend to prefer it instead of npm